config/application.rb

```diff
@@ -61,5 +61,30 @@ class Application < Rails::Application
 
     config.middleware.insert_before ActionDispatch::ParamsParser,
       HttpMethodNotAllowed
+
+    # rubocop:disable Metrics/LineLength
+    config.action_dispatch.default_headers = {
+      'X-Frame-Options'           => 'DENY',
+      'X-XSS-Protection'          => '1; mode=block',
+      'X-Content-Type-Options'    => 'nosniff',
+      'Strict-Transport-Security' =>  'max-age=31536000; includeSubDomains; preload',
+      'Content-Security-Policy'   => "default-src 'self'; " \
+                                     'font-src data:; '    \
+                                     "img-src 'self'; "    \
+                                     "object-src 'none'; " \
+                                     "script-src www.google-analytics.com 'self' 'unsafe-eval' " \
+                                     "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=' " + # govuk
+                                     "'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=' " + # govuk
+                                     "'sha256-9GTWoKmlaDM3V+GStWlXFaD4tf+wPfBN2ds2eySQ9aE=' " + # govuk
+                                     "'sha256-jS/ACiWphBsq3cSoeC2ouPyhfJqFFlLNOlwaOVN3UGc=' " + # javascript_i18n
+                                     "'sha256-4KTF3oamE52HQgyVL/chpyuwqC52Jk0kMg+KJiE+YlU=' " + # ga_page_view + @step_name = 'prisoner_step'
```

Comment:
If these hashes change is there a way that we'll be notified?
Have you looked if this step could be automated as part of of the build or middleware?